### PR TITLE
do reweight for balance the PGS of osds

### PR DESCRIPTION
**Pull request by liangzhaowang**
Signed-off-by: Zhaowangx.Liang <zhaowangx.liang@intel.com>

### CODE REVIEW

File: deploy/re_weight.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        self.max_time_re = 300
+        self.max_count_re = 1000
+        self.t_start = 0
+        self.t_end = 0
+        self.count_re = 0
+        self.result = []
+        self.isFitTarget = False
+
+    def Doreweight(self, oload, max_change, max_change_osds):
+        subprocess.Popen("ceph osd reweight-by-pg %s %s %s" % (oload, max_change, max_change_osds), shell=True, stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
```
Comment (xuechendi) [inline deploy/re_weight.py]: Don't do subprocess.Popen by you own, please use the one in common.py

File: workflow/workflow.py [hunk 2]
```diff
@@ -62,6 +62,7 @@ def run(self):
                     else:
                         run_deploy.main(['redeploy'])
                     tuner.main(['--section', section, 'apply_tuning'])
+                    re_weight.do()
```
Comment (xuechendi) [inline workflow/workflow.py]: is it a must do? You'd better just add this like a tuning option

**Comment by xuechendi** [comment]
@liangzhaowang , comment inline, also please give me some background about why you need to do this? Seems it should only to be a tuning? and the reweight args should not be hard coded.

**Comment by majianpeng** [comment]
@xuechendi . Now pg distribution not even. So we use 'osd reweight-by-pg' to make pg more even.  After creating a cluster do this to avoid move data.

**Pull request closed without merging**